Resources/Help/analyses/ANCOVA.md

```diff
@@ -0,0 +1,227 @@
+# ANCOVA 
+---
+ANCOVA allows the user to analyze the difference among three, or more, group means, when partialling out the effect of variables that have an influence on the dependent variable but are not part of the experimental manipulation (covariates). 
```

Comment:
.. difference between multiple group means, while adjusting for the effect... 
(i.e., covariates)